src/content/docs/extensions/json.mdx

```diff
@@ -515,4 +512,362 @@ and `"to"` in the file, whose values point to the primary key values of the unde
+Example 2:
+```cypher
+RETURN json_object("name","Alicia","age",28);
+```
+```json
+{"name":"Alicia","age":28}
+```
+
+### `json_array`
+
+Signature: `ARRAY -> JSON ARRAY`
+
+`json_array([any, ...])`
+
+Create a JSON array from `any` number of values.
+
+Example:
+
+```cypher
+RETURN json_array(42, 'kuzu', NULL);
+```
+```json
+[42,"kuzu",null]
+```
+
+
+### `json_merge_patch`
+
+Signature: `JSON, JSON -> JSON`
+
+`json_merge_patch(json, json)`
+
+Merges TWO `JSON` documents. Applies [RFC 7386](https://datatracker.ietf.org/doc/html/rfc7386)
+
+Example 1:
+
+```cypher
+RETURN json_merge_patch('{"name": "Alicia"}', '{"age": 28}');
+```
+```json
+{"name":"Alicia","age":28}
+```
+Example 2:
+```cypher
+RETURN json_merge_patch('3', NULL);
+```
+```json
+```
+Merging with a NULL path would result in NULL path.
+
+### `json_extract`
+
+Signatures: `JSON, STRING -> JSON`, `JSON, INTEGER -> JSON`, `JSON, LIST -> LIST of JSON`
+
+`json_extract(json, path)`
+
+Extracts `JSON` from `json` at the given `path`. 
+`path` is a `STRING` delimited by `'/'`.
+Integers may also be used to represent the path, which represents the index if the input is a JSON array.
+If `path` is a `LIST` of paths, the result will also be a `LIST`.
+
+If the path does not exist, this function returns an empty `JSON` document.
+
+Examples:
+
+In this case, we provide a single item as the `path`.
+
+```cypher
+RETURN json_extract('{"Software": {"Database": ["duck", "kuzu"]}}', 'Software/Database/1');
+```
+```json
+"kuzu"
+```
+
+Here, we provide a `LIST` of paths:
+
+```cypher
+RETURN json_extract('{"Software": {"Database": ["duck", "kuzu"]}}', ['Software/Database/1', 'Software/Database/0']);
+```
+```json
+["kuzu","duck"]
+```
+
+This example provides the path as an integer that represents the index of the item in the JSON array
+we want to extract:
+
+```cypher
+RETURN json_extract('[1, 2, 42]', 2)
+```
+```json
+42
+```
+
+Extracting from a nonexistent path results in empty `JSON` document:
+
+```cypher
+RETURN json_extract('{"Software": {"Database": ["duck", "kuzu"]}}');
+```
+```json
+```
+
+
+### `json_array_length`
+
+Signature: `JSON -> UINT32`
+
+`json_array_length(json[])`
+
+If the `json` is an JSON array, returns its length. Otherwise returns `0`.
+
+Example 1:
+
+```cypher
+return json_array_length('["1", "1", "4", null]');
+```
+```json
+4
+```
+Example 2:
+```cypher
+return json_array_length('{"kuzu": ["1", "1", "4", null]}');
+```
+```json
+0
+```
+(this is not an JSON array)
+
+### `json_contains`
+
+Signature: `JSON, JSON -> BOOL`
+
+`json_contains(json_haystack, json_needle)`
+
+Returns `True` if `json_needle` is contained in `json_haystack`. Both parameters are of `JSON` type, but `json_needle` can also be a numeric value or a `string`, however the `string` must be wrapped in double quotes.
+
+Example 1:
+
+```cypher
+RETURN JSON_CONTAINS('{"name": "Alicia"}', '"Alicia"');
+```
+```json
+True
+```
+Example 2:
+```cypher
+RETURN JSON_CONTAINS('{"age": 28}', '28');
```

Comment:
Can we also document that `'` is necessary here in the second parameter, or explain why that is necessary? I don't think this is intuitive for users to figure out.